src/editor/blocks/posts-inserter/index.js

```diff
@@ -38,12 +45,51 @@ const PostsInserterBlock = ( {
 	setInsertedPostsIds,
 	removeBlock,
 } ) => {
+	const [ ready, setReady ] = useState( ! attributes.displayFeaturedImage );
```

Comment:
Suggestion: There's a useful convention to prefix boolean variables with `is` (`isReady`, in this case)

Reply:
Thanks for the tip—updated in 0bbe397.